package.json

```diff
@@ -41,9 +41,14 @@
     "mocha": "^9.1.3",
     "semver": "^7.3.5",
     "sinon": "^11.1.2",
-    "sinon-chai": "^3.7.0"
+    "sinon-chai": "^3.7.0",
+    "typescript": "^4.4.4",
+    "@typescript-eslint/parser": "^5.3.0",
+    "@typescript-eslint/eslint-plugin": "^5.3.0"
   },
   "scripts": {
+    "build:ts": "tsc -b packages/*/tsconfig.build.json",
```

Comment:
This (may) result in unnecessary compiling. Basically every package in alphabetic order is built. For that, each of its dependency packages are built, and their dependency packages, some multiple times.

If possible (I know it might not be during conversion), this command should only build the main entry point, e.g.  `packages/controller/tsconfig.build.json`. `tsc -b` makes sure that the dependencies are built before that if necessary.

Reply:
I am not sure how I can optimize this.

Reply:
I don't think you can at the moment. Just wanted to put this out, I learned it the hard way when a simple change here cut the compile time from 40s to less than half in my project.